packages/access-client/src/agent.js

```diff
@@ -417,6 +422,7 @@ export class Agent {
    * @param {string} email
    * @param {object} [opts]
    * @param {AbortSignal} [opts.signal]
+   * @param {import('./types').ValidationPhraseHandler} [opts.onPhrase]
```

Comment:
I can't quite tell how I would decide whether I should provide a value for this and, if so, what a good value would be. Can you include a comment in the jsdoc and/or a test that makes it more clear why/how it's useful?

Reply:
yep good idea - fwiw here's an example:

https://github.com/web3-storage/w3ui/pull/385/files#diff-5ba6f9aada6d8184e7e8ee2fb08e2e6bfa67021b095c9a5603efd7b52b1dc3eeR120